### DISCUSSION TITLE
Collapsible sections in Automation editor please!

### DISCUSSION THREAD

**Discussion by NMBRPL8** [home-assistant/frontend#13156] [Automations / scripts / scenes]
Collapsible sections in Automation editor would make things much cleaner and faster to use in the UI. Especially with TriggerIDs being more common, combining what used to be several automations into one single automation is great, but they start to get extremely long - especially with the light.turn_on service giving such a loong list of options. Being able to collapse a section out of the way once done with it would be a huge help, especially when people are dealing with a 6 button controller which has 4 functions per button... The screenshot below only covers a couple of them, on mobile it takes forever to navigate, on computer it is not as bad but still not a good user experience when they get long Iike this, which they increasingly do now that the GUI automation builder is getting so powerful.

![Screenshot_20220709-081339_Home Assistant](https://user-images.githubusercontent.com/51841680/178084140-11a4098f-e07e-4f79-89db-05d3619a5701.jpg)

**Comment by karwosts** [comment]
This has been implemented for some time, closing.
